src/list/README.md

```diff
@@ -1,18 +1,18 @@
-# @dojo/widgets/menu
+# @dojo/widgets/list
```

Comment:
Should this talk about, or link to, resources?

Reply:
i can certainly raise a follow up to discuss etc